src/accsr/remote_storage.py

```diff
@@ -601,9 +601,9 @@ def get_pull_summary(
                 relative_obj_path = self._get_relative_remote_path(obj)
                 if not path_regex.match(relative_obj_path):
                     log.debug(f"Skipping {relative_obj_path} due to regex {path_regex}")
-                skip = True
+                    skip = True
 
-            else:
```

Comment:
`local_path` was not set in case regex matched. Leads to error later when trying to download.